app/controllers/transition_defence_requests_controller.rb

```diff
@@ -34,6 +34,7 @@ def transition_params
       defence_request: defence_request,
       transition_to: requested_transition,
       user: current_user,
+      auth_token: session[:user_token]
```

Comment:
The `current_user` is build from the `auth_token` on every request. And `current_user` is read only, so there's no reason we couldn't put it there. At least we'll have all user related information available in one place. But we definitely can do it later.